src/apiCalls.js

```diff
@@ -35,4 +40,4 @@ async function deleteBooking(bookingUpdate) {
 };
 
```

Comment:
Are you still working on getting it to update once you have "booked" a room? I love the idea of creating api functions that you can call in your other files.